client-tools/Backup/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+if (USE_ENTERPRISE)
+
+  if (MSVC AND NOT(SKIP_PACKAGING))
+    generate_product_version(ProductVersionFiles_arangobackup
+      NAME arangobackup
+      FILE_DESCRIPTION ${ARANGO_BACKUP_FRIENDLY_STRING}
+      ICON ${ARANGO_ICON}
+      VERSION_MAJOR ${CPACK_PACKAGE_VERSION_MAJOR}
+      VERSION_MINOR ${CPACK_PACKAGE_VERSION_MINOR}
+      VERSION_PATCH ${CPACK_PACKAGE_VERSION_PATCH}
+      VERSION_REVISION ${BUILD_ID}
+    )
+  endif ()
+
+  add_executable(${BIN_ARANGOBACKUP}
+    ${ProductVersionFiles_arangobackup}
+    ${CMAKE_SOURCE_DIR}/cmake/activeCodePage.manifest
+    BackupFeature.cpp
+    arangobackup.cpp
```

Comment:
Probably the `ADDITIONAL_BIN_ARANGOBACKUP_SOURCES` variable that was added here in the old CMakeLists did not exist, correct?